README.md

```diff
@@ -1,37 +1,117 @@
 
 # customs-financials-email-throttler
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Coverage](https://img.shields.io/badge/test_coverage-90-green.svg)](/target/scala-3.3.5/scoverage-report/index.html) [![Accessibility](https://img.shields.io/badge/WCAG2.2-AA-purple.svg)](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
+
 A backend service to control the rate at which emails are dispatched.
 
 This application lives in the "public" zone.
 
-## Running the application locally
+## Running the service
+
+*From the root directory*
+
+`sbt run` - starts the service locally
+
+`sbt runAllChecks` - Will run all checks required for a successful build
+
+Default service port on local - 9872
```

Comment:
Seems, there is no default port. Please check and update accordingly here and other places (if need)